### PR TITLE
[bug] Define index_sets, range_index_sets in outer chunk

### PR DESCRIPTION
**Pull request by sibaranov**
Solution to: https://sentry.summitlearning.org/lp/traject/issues/33515/

### CODE REVIEW

**Review by donaldong — COMMENTED** [review]
Interesting. I would expect the unit tests to catch this.

Let's add a test case covering this code path?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 require 'simplecov'
 require 'open3'
+require 'active_support/time'
```
Comment (donaldong) [inline spec/spec_helper.rb:4]: Is this used in the lib code also?

Comment (sibaranov) [reply spec/spec_helper.rb:4]: no, just for tests

**Review by donaldong — APPROVED** [review]
<a href="https://gitme.me/image?url=https%3A%2F%2Fcamo.githubusercontent.com%2Fbb2f786b3f9a7fe4c6982cc35187d59d6309f69c%2F687474703a2f2f692e696d6775722e636f6d2f70455566724c762e676966&token=lgtm" data-gitmeme-token="lgtm"></a>

**Pull request merged**